Apps/WebClient/src/NewClientApp/src/components/navmenu/BreadcrumbComponent.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import BreadcrumbItem from "@/models/breadcrumbItem";
+import { useAuthStore } from "@/stores/auth";
+import { useUserStore } from "@/stores/user";
+
+interface Props {
+    items?: BreadcrumbItem[];
+}
+const props = withDefaults(defineProps<Props>(), {
+    items: () => [],
+});
+
+const homeBreadcrumbItem: BreadcrumbItem = {
+    text: "Home",
+    to: "/home",
+    dataTestId: "breadcrumb-home",
+};
+
+const authStore = useAuthStore();
+const userStore = useUserStore();
+
+const isAuthenticated = computed(() => authStore.oidcIsAuthenticated);
+
+const isValidIdentityProvider = computed(
+    () => userStore.isValidIdentityProvider
+);
+const allBreadcrumbItems = computed(() => [homeBreadcrumbItem, ...props.items]);
+
+const displayBreadcrumbs = computed(
+    () => isAuthenticated.value && isValidIdentityProvider.value
+);
+</script>
+
+<template>
+    <v-breadcrumbs
+        v-if="displayBreadcrumbs"
+        data-testid="breadcrumbs"
+        class="pt-0 pb-0"
+        aria-label="Breadcrumb Nav"
+    >
+        <template v-for="(item, index) in allBreadcrumbItems" :key="item.text">
+            <v-breadcrumbs-item
+                :to="item.to"
```

Comment:
```suggestion
               :to="!item.active ? item.to : undefined"
```

Without this change, the currently displayed page will be a clickable link.